wsmaster/codenvy-hosted-api-license/src/main/java/com/codenvy/api/license/server/license/CodenvyLicenseFactory.java

```diff
@@ -33,19 +33,14 @@
 @Singleton
 public class CodenvyLicenseFactory {
 
+    protected static final String PRODUCT_ID = "OPL-STN-SM";
+
     private final char[] productId;
     private final String publicKey;
 
     @Inject
     public CodenvyLicenseFactory(@Named("license-manager.public_key") String publicKey) {
-        this.productId = "OPL-STN-SM".toCharArray();
-        this.publicKey = publicKey;
-    }
-
-    /** For testing purpose only */
-    @Deprecated
-    CodenvyLicenseFactory(char[] productId, String publicKey) {
-        this.productId = productId;
+        this.productId = PRODUCT_ID.toCharArray();
```

Comment:
PRODUCT_ID is a private data, so storing it in the String constant is a bad idea. Please, restore initial variant which is safe enough.